sdk-base/src/main/java/com/mapbox/maps/module/MapTelemetry.kt

```diff
@@ -23,8 +23,11 @@ interface MapTelemetry {
   fun disableTelemetrySession()
 
   /**
-   * Set the end-user selected state to participate or opt-out in telemetry collection.
+   * Set the debug logging enabled states.
```

Comment:
Maybe remove comment if it's deprecated and no-op?

Reply:
imo we shouldn't remove doc even its deprecated, but maybe state the API is deprecated in the API doc to make it clear. We also have checks to validate all the public APIs have documentation..